.github/workflows/build_demos.yml

```diff
@@ -0,0 +1,12 @@
+name: Build demos
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  run-check:
+    runs-on: windows-2022
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/demo/windows
```

Comment:
We have two separate workflows right now.
One runs for PRs and one for the man branch.

I think it makes sense if we build windows in both workflows:
* https://github.com/maplibre/maplibre-rs/blob/7371ae56fd1cf42cd1048cb024995a6228d7c2de/.github/workflows/on_main_push.yml
* https://github.com/maplibre/maplibre-rs/blob/7371ae56fd1cf42cd1048cb024995a6228d7c2de/.github/workflows/on_pull_request.yml

Reply:
I think it is a good idea. But artifacts take some storage space and I'm not sure about our limitations. The Windows artifact takes around 4MB I think. The MacOS artifact takes 80MB, probably because I don't build it in release mode yet.

Reply:
MacOS is huge because it includes maplibre-rs in quite a lot of different architectures for different platforms. So 80MB is totally fine. Its all release mode.

We don't have to worry about storage space. If it runs out we reduce the retention duraiton.

Reply:
I have moved the build action to the two workflows.